ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+DEFAULT_WORKDIR="/var/lib/jenkins"
+DEFAULT_KUBECONFIG_PATH=$DEFAULT_WORKDIR/kube.conf
```

Comment:
There is fetch_kubeconfig to get the kubeconfig, maybe this definition is no need anymore? Otherwise please explain what DEFAULT_KUBECONFIG_PATH is used for?

Reply:
this path stores the kubeconfig path whereas fetch_kubeconfig function is to export the fetched kube config to the kubeconfig path only

Reply:
> export KUBECONFIG=${KUBECONFIG_PATH}
But you already have KUBECONFIG to store the path, then can we remove the DEFAULT_KUBECONFIG_PATH? It seems you didn't use it in the new pipeline.